samples/gym-bipedal-walker-sample/README.md

```diff
@@ -0,0 +1,159 @@
+                    stump_height: number<0 .. 3>,
+                    obstacle_spacing: number<0 .. 6>,
+                }
+                training {
+                    LessonRewardThreshold: 230,
+                }
+            }
+```
+
+### No Curriculum Learning
+
+![](img/single-lesson.png)
+
+```javascript            
+            lesson hard_course {
+                scenario {
+                    stump_height: number<0 .. 3>,
+                    obstacle_spacing: number<0 .. 6>,
+                }
+            }
+
+```
+
+
+## Array Action
+
+| Action | Continuous Value | Array Elements |
+| ------ | ---------------- | ------- |
+| leg_1_torque[2] | [-1, 1] | [hip joint, knee joint] |
+| leg_2_torque[2] | [-1, 1] | [hip joint, knee joint] |
+
+## States
+
+Num   | Observation              |  Min   |   Max  | Mean
+------|--------------------------|--------|--------|------
+0     | hull_ang                 |  0     |  2*pi  |  0.5
+1     | hull_ang_vel             |  -inf  |  +inf  |  -
+2     | x_velocity               |  -1    |  +1    |  -
+3     | y_velocity               |  -1    |  +1    |  -
+4     | leg_1_hip_angle          |  -inf  |  +inf  |  -
+5     | leg_1_hip_speed          |  -inf  |  +inf  |  -
+6     | leg_1_knee_angle         |  -inf  |  +inf  |  -
+7     | leg_1_knee_speed         |  -inf  |  +inf  |  -
+8     | leg_1_contact            |  0     |  1     |  -
+9     | leg_2_hip_angle          |  -inf  |  +inf  |  -
+10    | leg_2_hip_speed          |  -inf  |  +inf  |  -
+11    | leg_2_knee_angle         |  -inf  |  +inf  |  -
+12    | leg_2_knee_speed         |  -inf  |  +inf  |  -
+13    | leg_2_contact            |  0     |  1     |  -
+14-23 | 10 lidar readings        |  -inf  |  +inf  |  -
+
+## Terminal Conditions
+
+- Episode length greater than 2000
+- Hull collides with ground or obstacle
+
+## Configuration Parameters
+
+| State    | Range                                             |
+| ---------| ------------------------------------------------------- |
+| stump_height | [0, 3] |
+| obstacle_spacing |  [0, 6] |
+
+## Install Requirements
+
+1. Download **either** [miniconda](https://conda.io/miniconda.html) or [Anaconda](https://www.anaconda.com/download/)
+2. Open Anaconda / miniconda command prompt
+3. Change directory to the parent directory
+    ```cmd
+    cd samples/gym-bipedal-walker-sample
+    pip install -r requirements.txt
+    ```
+
+## Running the Simulator Locally
+
+Run the simulator locally by the following command and setting environment variables for SIM_WORKSPACE and SIM_ACCESS_KEY.
+
+```bash
+python bipedal_walker.py
+```
+
+and then attach to your brain:
+
+```bash
+bonsai simulator unmanaged connect \                          
+    -b <brain_name> \
+    -a Train \
+    -c <concept_name> \
+    --simulator-name BipedalWalkerSim
+```
+
+Optionally, run the simulator without visualization:
+
+```bash
+python bipedal_walker.py --headless
+```
+
+## Evaluation
```

Comment:
hmm, we don't have support for running openai gym environments using `exported` brains in this repo. By run `main.py` are you referring to an MBA-gym sample somewhere?

Reply:
GOOD CATCH! I'm going to remove this for now in the README.